components/Home/sidebar/sidebar_items/BountiesSidebarItem.tsx

```diff
@@ -0,0 +1,118 @@
+import { css, StyleSheet } from "aphrodite";
+import { FEDocType } from "~/config/utils/getUnifiedDocType";
+import { ID, NullableString } from "~/config/types/root_types";
+import { ReactElement } from "react";
+import AuthorFacePile from "~/components/shared/AuthorFacePile";
+import colors from "~/config/themes/colors";
+
+type Props = {
+  bountyAmount: number;
+  bountyContentSnippet: string;
+  createdByAuthor: any;
+  documentType: FEDocType;
+  expirationDate: string;
+  isCommentBounty: boolean;
+  relatedDocID: ID;
+  slug: NullableString;
+};
+
+export default function BountiesSidebarItem({
+  bountyAmount,
+  bountyContentSnippet,
+  createdByAuthor,
+  documentType,
+  isCommentBounty,
+  relatedDocID,
+  slug,
+}: Props): ReactElement {
+  const roundedOfferAmount = bountyAmount;
+
+  return (
+    <div className={css(styles.bountiesSidebarItemContainer)}>
+      {/* NOTE: href is subject to change */}
+      <a
+        style={{ textDecoration: "none" }}
+        href={`/${
+          documentType === "question" ? "post" : documentType
```

Comment:
There is a method `getUrlToUniDoc` in routing.js which does this. I think you should tweak it a bit to inclue your use case

Reply:
Commented on this on `rightSidebarMigrate3`. 
The object payload from the BE doesn't necessarily have `UniDoc`.  May need to update BE or come up with another function.